src/core_modules/capture-core/components/Pages/MainPage/WorkingLists/epics/eventList.epics.js

```diff
@@ -9,7 +9,7 @@ import {
     deleteEventError,
     deleteEventSuccess,
 } from '../workingLists.actions';
-import { actionTypes as eventListActionTypes } from '../../EventsList';
+import { listViewActionTypes as eventListActionTypes } from '../../../../ListView';
```

Comment:
Maybe we can avoid the use of `as` here now that the action types have such a good distinct name ;)

Reply:
Removed in later PR.